source/common/http/filter_manager.h

```diff
@@ -36,6 +36,26 @@ class DownstreamFilterManager;
 
 struct ActiveStreamFilterBase;
 
+constexpr absl::string_view FS_LOCAL_REPLAY_KEY =
+    "envoy.filters.network.http_connection_manager.local_reply_owner";
+class LocalReplyOwnerType : public StreamInfo::FilterState::Object {
+public:
+  LocalReplyOwnerType(const std::string& filter_config_name)
+      : filter_config_name(filter_config_name) {}
+
+  ProtobufTypes::MessagePtr serializeAsProto() const override {
+    auto message = std::make_unique<ProtobufWkt::Struct>();
+    auto& fields = *message->mutable_fields();
+    *fields["filter_config_name"].mutable_string_value() = filter_config_name;
```

Comment:
Let's keep it simple and simply use a string value as serialized, the same as string serialization.

Reply:
you mean to keep the struct but with a string instead of map?

Reply:
Are we sure in the future this will not include other value?

Reply:
I'm not sure we won't need it, but that's OK since this is primarily for telemetry AFAICT, and is fairly free-form.